app/scripts/controllers/registration.js

```diff
@@ -37,26 +37,33 @@ angular
     $scope.savingAnswers = false;
 
     $scope.activePageId = pageId || '';
-    $scope.page = _.find(conference.registrationPages, { id: pageId });
-    $scope.activePageIndex = _.findIndex($scope.conference.registrationPages, {
-      id: pageId,
-    });
     // Filter through all pages and remove any empty pages
     // Check for registrant needed since on the welcome page
     // user could possibly not have started registration yet
     $scope.validPages =
       $scope.currentRegistration.registrants.length >= 1
         ? $scope.conference.registrationPages.filter(page => {
+            const registrantId = $scope.currentRegistrant
+              ? $scope.currentRegistrant
+              : $scope.currentRegistration.registrants[0].id;
```

Comment:
When the user presses ```Go back``` on the first page, they're navigated to ```http://eventregistrationtool.com/register/eventIdHere/page/``` which means ```$scope.currentRegistrant``` is ```undefined```. So we should check if it's truthy and if not just default to the first registrant's id.